blog/views.py

```diff
@@ -10,3 +11,18 @@ def home(request):
 
 def about(request):
     return render(request, "blog/about.html", {"title": "About"})
+
+@login_required
+def create_post(request):
+    if request.method == 'POST':
+        form=PostForm(request.POST)
+        if form.is_valid():
+            user=form.save(commit=False)
+            user.author=request.user
+            user.save()
+            messages.success(request, 'The post has been created successfully.')
+            return redirect('home')
+    else:
+        messages.error(request, 'Please correct the following errors:')
```

Comment:
The error message `Please correct the following errors` should be shown to the user only when there are any errors

![Error](https://github.com/Akash1362000/Django_BlogApp/assets/43296365/d4cb4a17-76d4-4000-8c53-b768cbde3aa9)

Reply:
Great, lemme work on it

Reply:
![Capture2](https://github.com/Akash1362000/Django_BlogApp/assets/143833326/5d346d7e-62eb-443e-92cf-e3829c38c0b3)
![Capture3](https://github.com/Akash1362000/Django_BlogApp/assets/143833326/7abcb9e6-8f26-486d-982c-438eac9b637a)